rpcs3/Emu/system_progress.cpp

```diff
@@ -236,7 +237,7 @@ void progress_dialog_server::operator()()
 						}
 					}
 
-					thread_ctrl::wait_for(10000);
+					thread_ctrl::wait_until(&sleep_until, 10000);
```

Comment:
there's a continue above that doesn't have any sleep.
Maybe we should move all the wait_until to the top of the loop